packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -733,8 +743,14 @@ export abstract class SharedObjectCore<
 }
 
 /**
- * SharedObject with simplified, synchronous summarization and GC.
- * DDS implementations with async and incremental summarization should extend SharedObjectCore directly instead.
+ * Helper for implementing {@link ISharedObject} with simplified, synchronous summarization and GC.
```

Comment:
Nit (I generally think we should avoid abbreviating "Garbage Collection" for accessibility)
```suggestion
 * Helper for implementing {@link ISharedObject} with simplified, synchronous summarization and garbage collection.
```